src/engine/profiler.js

```diff
@@ -126,6 +132,12 @@ class Profiler {
          */
         this.records = [];
 
+        this.increments = [];
```

Comment:
Could you add comments describing what `increments` and `counters` are?

Reply:
Added,